onnxruntime/python/tools/transformers/microbench/benchmark.py

```diff
@@ -0,0 +1,30 @@
+import time
```

Comment:
the microbench is not related with transformers, right? consider move it to a separate location.

Reply:
Update: moved to tools directory.